dotCMS/src/main/java/com/dotmarketing/common/util/SQLUtil.java

```diff
@@ -237,6 +240,64 @@ public static String sanitizeSortBy(String parameter){
 		SecurityLogger.logDebug(SQLUtil.class, "Invalid or pernicious sql parameter passed in : " + parameter);
 		return StringPool.BLANK;
 	}
+
+	/**
+	 * Method to translate UI field into SQL columns naming
+	 * @param parameter
+	 * @return
+	 */
+	public  static String translateSortBy(String parameter){
+
+		String result = "";
+		result = parameter.replace("moddate","mod_date")
+				.replace("categoryname","category_name")
+				.replace("categoryvelocityvarname","category_velocity_var_name")
+				.replace("categorykey","category_key")
+				.replace("pageurl", "page_url")
+				.replace("velocityvarname","velocity_var_name")
+				.replace("sortorder","sort_order")
+				.replace("hostname","hostName")
+				.replace("relationtypevalue","relation_type_value")
+				.replace("childrelationname","child_relation_name")
+				.replace("parentrelationname","parent_relation_name");
+
+			return  result;
+	}
+
+	/**
+	 * Method to convert camel case convention into snake case
+	 * @param parameter
+	 * @return
+	 */
+	public static String convertCamelToSnake(String parameter)

```

Comment:
Please move this method over to the `com.dotmarketing.util.StringUtils` class, which is a Util class meant for String-related operations.

Reply:
This feedback has been addressed @jcastro-dotcms